spec/wavesurfer.spec.js

```diff
@@ -357,6 +357,46 @@ describe('WaveSurfer/playback:', function() {
         });
         wavesurfer.destroy();
     });
+
+    fdescribe('seekTo: event emission', function() {
+        var interactionEventSpy;
+        var seekEventSpy;
+
+        beforeEach(function() {
+            interactionEventSpy = jasmine.createSpy();
+            seekEventSpy = jasmine.createSpy();
+
+            wavesurfer.on('interaction', function() {
```

Comment:
these event listeners need to be removed at some point no?

Reply:
Ah so you were thinking it would just disable the seek event? I guess that works, was thinking more broadly that it would disable all events, which now that I’m thinking about it seems less than ideal.

Reply:
sure I will remove them in an afterEach block